test/sequential/test-process-constrained-memory.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const { Worker } = require('worker_threads');
+
+if (!process.env.isWorker) {
+  process.env.isWorker = true;
+  new Worker(__filename);
+  assert(process.constrainedMemory() >= 0);
+} else {
+  assert(process.constrainedMemory() >= 0);
```

Comment:
It might be worth adding a check that `process.constrainedMemory() <= os.totalmem()`

Reply:
From the Libuv documentation.  [`uv_get_constrained_memory` ](https://docs.libuv.org/en/v1.x/misc.html#c.uv_get_constrained_memory) can be less than or greater than [`uv_get_total_memory()`](https://docs.libuv.org/en/v1.x/misc.html#c.uv_get_total_memory).

Reply:
Ugh ok